app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="activity_main_guideline_begin">60dp</dimen>
+    <dimen name="activity_main_guideline_title">120dp</dimen>
+    <dimen name="activity_main_marginBottom_result">50dp</dimen>
+    <dimen name="activity_main_textSize_result">40sp</dimen>
+    <dimen name="activity_main_textSize_title">30sp</dimen>
+    <dimen name="activity_main_marginBottom_button">15dp</dimen>
+    <dimen name="activity_main_size_icon">70dp</dimen>
+    <dimen name="activity_main_margin_start_icon">15dp</dimen>
```

Comment:
please fix the dimens names. Words un lower case separated by an underscore "_"
activity_main_marginBottom_result --> activity_main_margin_bottom_result

Reply:
Solved!